packages/artifact/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@actions/artifact",
+  "version": "0.1.0",
+  "preview": true,
+  "description": "Actions artifact lib",
+  "keywords": [
+    "github",
+    "actions",
+    "artifact"
+  ],
+  "homepage": "https://github.com/actions/toolkit/tree/master/packages/artifact",
+  "license": "MIT",
+  "main": "lib/artifact.js",
+  "types": "lib/artifact.d.ts",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/actions/toolkit.git",
+    "directory": "packages/artifact"
+  },
+  "scripts": {
+    "audit-moderate": "npm install && npm audit --audit-level=moderate",
+    "test": "echo \"Error: run tests from root\" && exit 1",
+    "tsc": "tsc"
+  },
+  "bugs": {
+    "url": "https://github.com/actions/toolkit/issues"
+  },
+  "dependencies": {
+    "@actions/core": "^1.2.1",
+    "@actions/http-client": "^1.0.1"
```

Comment:
We should use the version that supports proxy.  Why not use the latest `http-client@1.0.6`?

Reply:
Updated, to `1.0.6`. When I initially installed it, `1.0.1` was the most recent (even this old version does support proxy though). Before I was using the `typed-rest-client` npm package